Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -521,7 +521,8 @@ def is_linear(self):
         return self.linear_flag
 
     def calculate_norm(self):
-        '''Returns the norm of the SumOperator'''
+        '''Returns the norm of the SumOperator. Note that this gives a NotImplementedError if the SumOperator is not linear.'''
+
         if self.is_linear():
             return LinearOperator.calculate_norm(self)

```

Comment:
looks to me that these lines should sit in `Operator` already. There's nothing here that is specific to `SumOperator`